git_machete/client.py

```diff
@@ -937,12 +937,12 @@ def fp_hash(branch_: LocalBranchShortName) -> Optional[FullCommitHash]:
         maybe_space_before_branch_name = ' ' if self.__git.get_boolean_config_attr('machete.status.extraSpaceBeforeBranchName') else ''
 
         def print_line_prefix(branch_: LocalBranchShortName, suffix: str) -> None:
-            out.write("  " + indent_before_branch_name)
+            out.write("  " + maybe_space_before_branch_name)
```

Comment:
Also! if you could also add a test for this config, preferably in `test_status.py`

Even though tests run in ASCII-only mode, this feature applies to both ASCII-only mode and "colorful" mode with non-ASCII edges & ANSI-escape colors

Reply:
sure